metrics/integration.go

```diff
@@ -45,7 +45,7 @@ var (
 		prometheus.HistogramOpts{
 			Name:    "integration_svc_snapshot_created_to_pipelinerun_with_ephemeral_env_started_seconds",
 			Help:    "The duration measures the time elapsed between the creation of a snapshot resource and the initiation of an integration PipelineRun for pipelines operating within an ephemeral environment.",
-			Buckets: []float64{0.05, 0.1, 0.5, 1, 2, 3, 4, 5, 10, 15, 30},
+			Buckets: []float64{1, 3, 5, 10, 15, 20, 25, 30, 35, 45, 60},
```

Comment:
I still think we should include buckets up to 300s limit
```suggestion
			Buckets: []float64{1, 5, 10, 15, 30, 45, 60, 120, 240, 300},
```

Reply:
We can do that, but I want to keep in the 20, 25, and 35s buckets.  Most of the ephemeral environments have taken 15-30s so far and I'd like to know a bit more about what's going on in there.  For example, it's possible that we had 5 provisionings take 29s and 1 take 31s but its also possible we had 5 take 16s and 1 take 100s. There's no reason we need to have 11 buckets, right?

Reply:
yeah, fine by me, I just wanted to extend it to full period